keycloak/themes/tbpro/assets/vue/App.vue

```diff
@@ -1,164 +1,17 @@
 <script setup lang="ts">
-import { ref } from "vue";
-import OrbGraphic from '@/images/orb-graphic.png';
-import router from "../router";
+import { ref } from 'vue';
+import router from '../router';
+import BaseTemplate from '@kc/vue/BaseTemplate.vue';
 
 // Match routes based on pageId / route name
 const pageId = ref(window._page.pageId);
 const routeName = pageId.value && router.hasRoute(pageId.value) ? pageId.value : 'route-not-implemented';
 console.log('route:', routeName);
-router.replace({name: routeName});
-
-
-const playingAnimation = ref(false);
-const onWiggle = async () => {
-  playingAnimation.value = true;
-  window.setTimeout(() => {
-    playingAnimation.value = false;
-  }, 1000);
-};
-
+router.replace({ name: routeName });
 </script>
 
 <template>
-  <section>
-    <div class="debug-page-id" @click="onWiggle" :class="{wiggle: playingAnimation}">{{ pageId }}</div>
-
-    <div class="card">
-      <!-- Left side: Orb graphic -->
-      <div class="left-side">
-        <img :src="OrbGraphic" alt="Thunderbird Orb" class="orb-graphic" />
-      </div>
-
-      <!-- Right side: Panel -->
-      <div class="right-side">
-        <div class="panel">
-          <router-view />
-        </div>
-      </div>
-    </div>
-  </section>
+  <base-template>
```

Comment:
Simplified so we can re-use the base template without having to refactor the App.vue logic.